apps/emqx_management/src/proto/emqx_management_proto_v5.erl

```diff
@@ -0,0 +1,96 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2022-2023 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+
+-module(emqx_management_proto_v5).
+
+-behaviour(emqx_bpapi).
+
+-export([
+    introduced_in/0,
+
+    node_info/1,
+    broker_info/1,
+    list_subscriptions/1,
+
+    list_listeners/1,
+    subscribe/3,
+    unsubscribe/3,
+    unsubscribe_batch/3,
+
+    call_client/3,
+
+    get_full_config/1,
+    get_full_config_v2/1,
+    get_config_v2/2,
```

Comment:
Do we really need to introduce another layer of versioning here? Are there any clients of `get_full_config/1` left? If not, we should just drop it, this is a new proto version anyway.

Reply:
yes, get_full_config/1 is using here(will deprecated when 5.2.0, but keep in 5.1.x):
https://github.com/zhongwencool/emqx/blob/fix-get-config-from-target-node/apps/emqx_management/src/emqx_mgmt_api_configs.erl#L375-L379
get_full_config_v2/1 is using here:
https://github.com/zhongwencool/emqx/blob/fix-get-config-from-target-node/apps/emqx_management/src/emqx_mgmt_api_configs.erl#L391-L398

Reply:
Ah, I see, thanks for clarifying that. I'd then propose 2 things:
* Avoid making another application's CLI module the target of RPC, I believe it's much better to keep the target in the same module, or at least in the same application, even if it will amount to copypasting some of the code. Middle ground would be moving the target of RPC to the `emqx_conf` module interface and marking it as such, for example with a comment.
* Give those functions a bit more descriptive names instead of `v1` / `v2`, so that it will be easier to understand what's the difference between them.

Reply:
done